Fluentbit: docs: Update input concept for style (#1485)
fluent/fluent-bit-docs: concepts/data-pipeline/input.md

# Input

[Fluent Bit](http://fluentbit.io) provides input plugins to gather information from
different sources. Some plugins collect data from log files, while others can

> gather metrics information from the operating system. There are many plugins for

Reviewer: ```suggestion
gather metrics information from the operating system. There are many plugins to suit
```

    style A stroke:darkred,stroke-width:2px;
```

When an input plugin loads, an internal _instance_ is created. Each instance has its
own independent configuration. Configuration keys are often called _properties_.

> Every input plugin has its own documentation section where it's specified how it can
> be used and what properties are available.

Reviewer: ```suggestion
Every input plugin has its own documentation section that specifies how to use it
and what properties are available.
```
